rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
+end
```

Comment:
- [ ] Add the association of a rental with both the book class and the person class. Remember that a rental belongs to both a book and a person.